homeassistant/helpers/storage.py

```diff
@@ -88,6 +92,91 @@ def load_old_config():
     return config
 
 
+def get_store_manager(
+    hass: HomeAssistant, config_dir: str | None = None
```

Comment:
Why pass config dir but store in single key?

Reply:
Creating HomeAssistant creates a Storage object for the `ConfigStore` so we need to pass it because `hass.config` won't be created on the `hass` object yet